### ISSUE TITLE
eslint 2.0

### ISSUE THREAD

**Issue by demrks** [jamespamplin/eslint-config-idiomatic#3]
eslint 2.0 renamed some rules:

```
1:1   error  Rule 'no-empty-label' was removed and replaced by: no-labels                 no-empty-label
1:1   error  Rule 'space-after-keywords' was removed and replaced by: keyword-spacing     space-after-keywords
1:1   error  Rule 'space-return-throw-case' was removed and replaced by: keyword-spacing  space-return-throw-cas
```

**Comment by jamespamplin** [comment]
Thanks @demrks - working on supporting ESLint 2.0. Should have a release out shortly.

**Comment by jamespamplin** [comment]
Published `eslint-config-idiomatic@2.0.0` with ESLint 2.0 support.

@demrks please let me know if you have any issues.

**Comment by demrks** [comment]
@jamespamplin Thanks, works for me.
